docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,200 @@
+S3OutputModule
+==============
+
+### About Amazon S3
+-------------------
+
+**Amazon S3 (Simple Storage Service)** is an object storage system with a web service interface to store and retrieve any amount of data at any time from anywhere on the web, offered by Amazon Web Services.
+
+### Overview
+------------
+
+S3OutputModule is to upload files/directories into an S3. Parallelism of uploading files into S3 can be achieved by partitioning the AbstractFileOutputOperator.
+But, S3OutputModule is needed to parallel uploading the blocks of a file into S3. 
+
+Below table illustrates the benefits of this module over AbstractFileOutputOperator.
+
+| S3OutputModule                                                                                                                                                                           | AbstractFileOutputOperator                                                                                                                                                                                                                           |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Maximum file size for upload can be 5TB.                                                                                                                                                 | Maximum file size for upload can be 5GB.                                                                                                                                                                                                             |
```

Comment:
Maximum file size for upload can be 5TB => Maximum upload file size is 5TB
Maximum file size for upload can be 5GB. => Maximum upload file size is 5GB.